### PR TITLE
EDU-480 Unify rooms tab (and more)

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-480

### CODE REVIEW

File: src/styles/info-cell.less [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
   color: fade(@edu-text-color, 80%);
 }
 
-.DocumentInfoCell-dates {
+.InfoCell-subtext {
   margin-top: 0.75em;
   font-size: 0.9em;
   color: @edu-text-color-secondary;
-}
+}
```
Comment (ahelmberger) [inline src/styles/info-cell.less:21]: You seem to have removed the final new line character (also in the other LESS file), maybe you can check if your editor picks up the `.editorconfig` file correctly.

File: src/components/favorites-tab.yml [hunk 2]
```diff
@@ -1,18 +1,9 @@
...
-  de: Keine Unterrichtseinheiten as Favorit hinzugefügt
+set:
+  en: Added
+  de: Hinzugefügt
+setOn:
+  en: Added on
+  de: Hinzugefügt am
+sortedBySetOn:
+  en: Sorted by date of adding
+  de: Nach Hinfügungsdatum sortiert
```
Comment (ahelmberger) [inline src/components/favorites-tab.yml:9]: "Hinzufügungsdatum", unfortunately :laughing:

**Comment by ahelmberger** [comment]
Thanks for going the extra mile and making all tables on the page work perfectly including the sorting :heart:

File: src/styles/info-cell.less (hunk 1 above)
Comment (adrianaluchian) [reply src/styles/info-cell.less]: Must have been a strange glitch, I just opened and re-saved the files now and the line was added. Updated both!

File: src/components/favorites-tab.yml (hunk 2 above)
Comment (adrianaluchian) [reply src/components/favorites-tab.yml]: Thanks, updated! 😁

**Comment by adrianaluchian** [comment]
> Thanks for going the extra mile and making all tables on the page work perfectly including the sorting ❤️

Thank you! ☺️

**Pull request merged**